src/main/resources/db/changelog/changes/create-shopping_cart-table.yaml

```diff
@@ -0,0 +1,26 @@
+databaseChangeLog:
+  - changeSet:
+      id: create-shopping_cart-table
+      author: YEvhenii
+      changes:
+        - createTable:
+            tableName: shopping_cart
+            columns:
+              - column:
+                  name: id
+                  type: bigint
+                  autoIncrement: true
+                  constraints:
+                    nullable: false
+                    primaryKey: true
+              - column:
+                  name: user_id
```

Comment:
where is foreign key to the user table?

Reply:
https://mateacademy.slack.com/archives/C05B8U42C69/p1694610657937319